.github/workflows/linter.yml

```diff
@@ -26,4 +26,4 @@ jobs:
         pip install flake8
     - name: Lint utilities
       run: |
-        flake8 fink_utils/*/*.py --count --show-source --statistics --ignore=E302,E501
+        flake8 ../fink-utils
```

Comment:
Why do you need to change the path?

Reply:
I added a .flake8 file that specifies the flake8 options without putting each option in the command line, but you need to run flake8 from the project root folder. flake8 will discover the .flake8 file and parse it.

Reply:
flake8 is also able to discover each .py files. So, we can remove the /*/*.py glob.

Reply:
OK!